src/server/scripts/Kalimdor/CavernsOfTime/CullingOfStratholme/boss_infinite.cpp

```diff
@@ -74,20 +74,27 @@ class boss_infinite_corruptor : public CreatureScript
         {
             Talk(SAY_DEATH);
             for (SummonList::const_iterator itr = summons.begin(); itr != summons.end(); ++itr)
+            {
                 if (Creature* cr = ObjectAccessor::GetCreature(*me, (*itr)))
                 {
                     if (cr->GetEntry() == NPC_TIME_RIFT)
+                    {
                         cr->DespawnOrUnsummon(1000);
+                    }
                     else
                     {
                         cr->DespawnOrUnsummon(5000);
                         cr->RemoveAllAuras();
                         cr->MonsterSay("You have my thanks for saving my existence in this timeline. Now i must report back to my superiors. They must know immediately of what i just experienced.", LANG_UNIVERSAL, 0);
                     }
                 }
+            }
 
             if (InstanceScript* pInstance = me->GetInstanceScript())
+            {
                 pInstance->SetData(DATA_SHOW_INFINITE_TIMER, 0);
+                pInstance->DoRemoveAurasDueToSpellOnPlayers(60588);
```

Comment:
shall we move `60588` into a constant? so the code will also be more readable :)

Reply:
Yes please. 
It already exists in the `enum Spells` on line 10

SPELL_CORRUPTING_BLIGHT

Reply:
```suggestion
                pInstance->DoRemoveAurasDueToSpellOnPlayers(SPELL_CORRUPTING_BLIGHT);
```

Reply:
weird... I thought I have placed the name instead of the Spell ID, ups !  but yeah that was the intention since the beggining. 🐱